README.md

```diff
@@ -1,2 +1,4 @@
 # hello-world
 ideas and practice
+
+Today is a really hard day. All I can think about is Whitley and how I don't know when I will ever see her again. Also Tommy and why he said he missed me but won't talk to me. It is hard to breathe. I can't stop crying. I'm never dating or doing this ever again. Fuck this noise. I'll have pets and be fine. I feel like I'm dying.
```

Comment:
Everyone says it will get better. And I know that in some ways it will. But I also need people to realize that something in this experience changed me, changed who I am, and what I want, and what/who I need. There are parts of me right now that are literally dying. They will not return until she returns. End of story.